src/test/java/systemtests/SortCommandSystemTest.java

```diff
@@ -0,0 +1,147 @@
+package systemtests;
+
+import static seedu.saveit.model.issue.IssueSort.CHRONOLOGICAL_SORT;
+import static seedu.saveit.model.issue.IssueSort.FREQUENCY_SORT;
+import static seedu.saveit.model.issue.IssueSort.TAG_SORT;
+import static seedu.saveit.testutil.TypicalIssues.ALICE;
+import static seedu.saveit.testutil.TypicalIssues.BENSON;
+import static seedu.saveit.testutil.TypicalIssues.CARL;
+import static seedu.saveit.testutil.TypicalIssues.DANIEL;
+import static seedu.saveit.testutil.TypicalIssues.ELLE;
+import static seedu.saveit.testutil.TypicalIssues.FIONA;
+import static seedu.saveit.testutil.TypicalIssues.GEORGE;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import seedu.saveit.commons.core.Messages;
+import seedu.saveit.logic.commands.FindCommand;
+import seedu.saveit.logic.commands.ListCommand;
+import seedu.saveit.logic.commands.SortCommand;
+import seedu.saveit.model.Issue;
+import seedu.saveit.model.Model;
+import seedu.saveit.model.issue.IssueSort;
+
+public class SortCommandSystemTest extends SaveItSystemTest {
+    @Test
+    @Ignore
+    public void sort() {
+        Model expectedModel = getModel();
+
+        /* Case: sort issues by tag sort in saveit book, command with leading spaces and trailing spaces */
+        String command = "   " + SortCommand.COMMAND_WORD + " " + TAG_SORT + "   ";
+        ModelHelper.setSortedList(expectedModel, DANIEL, BENSON, ALICE, CARL, ELLE, FIONA, GEORGE);
+        assertCommandSuccess(command, IssueSort.TAG, expectedModel);
+        assertSelectedCardUnchanged();
```

Comment:
Currently this assertion may not be working as no card will be selected at root directory.

Reply:
Noted with thanks. will update soon